eng/helix/content/RunTests/RunTests.csproj

```diff
@@ -4,6 +4,9 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net7.0</TargetFramework>
     <DefineConstants Condition=" '$(InstallPlaywright)' == 'true' ">$(DefineConstants);INSTALLPLAYWRIGHT</DefineConstants>
+    <NoWarn>$(NoWarn);CA2007;NU5104</NoWarn>
+    <IsUnitTestProject>false</IsUnitTestProject>
+    <IsTestProject>false</IsTestProject>
```

Comment:
Arcade doesn't work the way I'd hoped and overrides `$(IsTestProject)`. Instead of this and `Sdk="Microsoft.NET.Sdk"` in the `<Project>` tag:

``` msbuild
  <PropertyGroup>
    <IsUnitTestProject Condition=" '$(SkipTestBuild)' != 'true' ">true</IsUnitTestProject>
  </PropertyGroup>

  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Will also need `<Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />` near the bottom of the file.

Reply:
94105ca should handle that.